### PR TITLE
Components update

### PR DESCRIPTION
**Pull request by tcrst**
- Fixing Grafana dashboards that were broken since Gimlet Stack v0.5.0

  - Ingress Nginx  update from 0.49 to 1.0.2

  - Grafana update from 8.0.3 to 8.1.5

  - Prometheus helm chart is updated to 14.7.1
    Just a regular cadence of version updates; including fixes, features, etc

  - Fixed Prometheus Dashboard and updated Nginx dashboard to newer version

### CODE REVIEW

File: examples/stack-full.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
+---
```
Comment (laszlocph) [inline examples/stack-full.yaml:1]: Is there any functional change in this file? If not, can you remove it from the changeset?

File: examples/stack-full.yaml [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 stack:
-  repository: "https://github.com/gimlet-io/gimlet-stack-reference.git"
+  repository: https://github.com/tcrst/gimlet-stack-reference.git?tag=v0.6.0
```
Comment (laszlocph) [inline examples/stack-full.yaml:3]: The file should not point to your fork.

File: examples/stack-full.yaml [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+    user: githubUser
+  prometheus:
     enabled: true
-    org: "My Company"
-    user: "githubUser"
-    clientID: "xxx"
-    clientSecret: "xxx"
   sealedSecrets:
-    sealingKey: "xxx"
+    sealingKey: xxx
```
Comment (laszlocph) [inline examples/stack-full.yaml:21]: I could be wrong, but yaml strings are safest to be quoted I think.

File: manifests/grafana-dashboard-nginx.yaml [hunk 4]
```diff
@@ -19,6 +19,12 @@ data:
             "hide": true,
```
Comment (laszlocph) [inline manifests/grafana-dashboard-nginx.yaml:19]: Can you revert this file to the original one?

The dashboard works for me, and the change in the file is difficult to follow.

**Review by laszlocph — CHANGES_REQUESTED** [review]
Thank you for the change!

Especially for fixing the Prometheus dashboard!

File: examples/stack-full.yaml (hunk 2 above)
Comment (tcrst) [reply examples/stack-full.yaml]: Removed

File: examples/stack-full.yaml (hunk 1 above)
Comment (tcrst) [reply examples/stack-full.yaml]: Curious how that got there

File: examples/stack-full.yaml (hunk 3 above)
Comment (tcrst) [reply examples/stack-full.yaml]: Reverted to the initial one

File: manifests/grafana-dashboard-nginx.yaml (hunk 4 above)
Comment (tcrst) [reply manifests/grafana-dashboard-nginx.yaml]: Done

**Review by tcrst — COMMENTED** [review]
Should be fine now

**Pull request merged**